tensorflow/python/ops/math_ops.py

```diff
@@ -1381,7 +1427,7 @@ def equal(x, y, name=None):
   boolean values.
 
   For example:
-
+  
```

Comment:
Now this looks like a line you changed. Please configure your editor to remove trailing whitespace on save or remove it manually.
![LAEO9bhS4sW](https://user-images.githubusercontent.com/323199/72914889-407f6900-3cf4-11ea-9bb2-10f2bef4a712.png)

Reply:
Trailing whitespace not removed